build-logic/src/main/kotlin/geyser.modded-conventions.gradle.kts

```diff
@@ -93,24 +92,24 @@ afterEvaluate {
 
     // These are shaded, no need to JiJ them
     configurations["shadow"].dependencies.forEach {shadowed ->
-        println("Not including shadowed dependency: ${shadowed.group}:${shadowed.name}")
```

Comment:
Is there any reason this is commented out (and the ones below)?

Reply:
I'm fine with this unless there's a way we either explicitly request a debug mode through Gradle, or if we can sidestep running this unless we requested a Fabric/NeoForge build.

Reply:
Resolved on discord.